test/src/unit-QueryCondition.cc

```diff
@@ -173,6 +174,825 @@ TEST_CASE("QueryCondition: Test char", "[QueryCondition][char_value]") {
   REQUIRE(query_condition.field_names().count(field_name) == 1);
 }
 
+TEST_CASE(
```

Comment:
No, this is because we test that the AST functions work in unit-query-ast.cc and then we test that we can call the functions properly from the QueryCondition interface in unit-QueryCondition.cc.